lib/plugins/extension/helper/extension.php

```diff
@@ -293,8 +328,10 @@ public function getEmailID()
      */
     public function getDescription()
     {
-        if (!empty($this->localInfo['desc'])) return $this->localInfo['desc'];
-        if (!empty($this->remoteInfo['description'])) return $this->remoteInfo['description'];
+        if (!empty($this->localInfo['desc']) && !$this->isIgnoredSource('local'))
+            return $this->localInfo['desc'];
```

Comment:
For statements that no longer fit into one line, I would prefer parentheses.

Reply:
Basically I don't have any automatic formatting tool in my dev environment and I was tired of so many attributes (so I probably used find-and-replace for this). Will fix them though.

Reply:
We have access to free licenses for JetBrain's IntelliJ Idea which is an excellent IDE for PHP and lots of other languages. It makes refactoring, code formatting etc. a breeze. I can hook you up with a license once JetBrains renews our OpenSource membership for this year.